packages/web/src/pages/modals/Modals.tsx

```diff
@@ -31,7 +31,6 @@ import { TransactionDetailsModal } from 'components/transaction-details-modal'
 import UnfollowConfirmationModal from 'components/unfollow-confirmation-modal/UnfollowConfirmationModal'
 import UnloadDialog from 'components/unload-dialog/UnloadDialog'
 import { UploadConfirmationModal } from 'components/upload-confirmation-modal'
-import { USDCManualTransferModal } from 'components/usdc-manual-transfer-modal/USDCManualTransferModal'
```

Comment:
:rip:

Reply:
might even get rid of it in mobile later.